app/models/teacher_set.rb

```diff
@@ -789,4 +789,10 @@ def update_notes(teacher_set_notes_string)
       TeacherSetNote.create(teacher_set_id: self.id, content: note_content)
     end
   end
+
+  #Updating available count, total count and availability_string
+  def update_available_and_total_count(total_ct, available_ct)
+    availability_string = (available_ct.to_i > 0) ?  AVAILABILITY_LABELS['available']  : AVAILABILITY_LABELS['unavailable'] 
```

Comment:
The current values in the database are "available" and "unavailable".  I think the "Available" and "Checked Out" are used on the front end.  If you'd like to be calling a constant here, you'll need to make another one, or use the AVAILABILITY_LABELS keys.